plugins/module_utils/prism/clusters.py

```diff
@@ -5,13 +5,104 @@
 __metaclass__ = type
 
 from .prism import Prism
+from copy import deepcopy
 
 
 class Cluster(Prism):
     def __init__(self, module):
         resource_type = "/clusters"
         super(Cluster, self).__init__(module, resource_type=resource_type)
+        self.build_spec_methods = {
+            "authorized_public_key_list": self._build_spec_public_keys,
+            "timezone": self._build_spec_timezone,
+            "supported_information_verbosity": self._build_spec_supported_information_verbosity,
+            "redundancy_factor": self._build_spec_redundancy_factor,
+            "network": self._build_spec_network,
+        }
 
+    def _get_default_spec(self):
+        return deepcopy(
+            {
+                "api_version": "3.1.0",
+                "metadata": {"kind": "cluster", "spec_version": 2},
```

Comment:
Spec versions is not required, we generally use metadata coming from get API when there is only update functionality